mne/viz/backends/_pyvista.py

```diff
@@ -874,20 +874,21 @@ def _process_events(plotter):
             plotter.app.processEvents()
 
 
-def _set_colormap_range(actor, ctable, scalar_bar, rng=None):
+def _set_colormap_range(actor, ctable, scalar_bar, rng=None,
+                        background_color=None):
     from vtk.util.numpy_support import numpy_to_vtk
-    mapper = actor.GetMapper()
-    lut = mapper.GetLookupTable()
-    # Catch:  FutureWarning: Conversion of the second argument of
-    # issubdtype from `complex` to `np.complexfloating` is deprecated.
-    with warnings.catch_warnings():
-        warnings.filterwarnings("ignore", category=FutureWarning)
-        lut.SetTable(numpy_to_vtk(ctable))
     if rng is not None:
-        mapper.SetScalarRange(rng[0], rng[1])
-        lut.SetRange(rng[0], rng[1])
+        actor.GetMapper().SetScalarRange(*rng)
     if scalar_bar is not None:
-        scalar_bar.SetLookupTable(actor.GetMapper().GetLookupTable())
+        lut = scalar_bar.GetLookupTable()
+        if background_color is not None:
+            background_color = np.array(background_color) * 255
+            alphas = ctable[:, -1][:, np.newaxis] / 255.
+            use_table = ctable.copy()
+            use_table[:, -1] = 255.
+            ctable = (use_table * alphas) + background_color * (1 - alphas)
```

Comment:
On second thought, can you DRY this code with something like `ctable = _alpha_blend_background(ctable, background_color)`, maybe in `mne/viz/_3d.py` that you can do a nested import in these two places?
```
mne/viz/backends/_pysurfer_mayavi.py
mne/viz/backends/_pyvista.py
```
`git grep "alphas = "` showed me that these code paths are probably duplicated...